### ISSUE TITLE
Change zise Animate tool in opcion "All"

### ISSUE THREAD

**Issue by zarco1794** [opentoonz/opentoonz#2852]
the image size to Animate tool in the opción "All" it s sou small, take control is very dificulty. maybe it would be better to increase the size of the "animate tool All" icon
![1](https://user-images.githubusercontent.com/39812147/67245083-898ffa80-f431-11e9-9829-5f324c28cfe8.JPG)
It is very difficult to use the "Mass" option
https://media.giphy.com/media/Wt0sKUnXqFl5GNTqTZ/giphy.gif 

![2](https://user-images.githubusercontent.com/39812147/67246467-e80aa800-f434-11e9-9fc3-10ef47d70ec8.JPG)

**Comment by beeheemooth** [comment]
I agree with you: the manipulator is really very small. In addition, despite the fact that it is bright in color, it is sometimes difficult to find.

**Comment by gab3d** [comment]
Other software usually have a way for the user to adjust the manipulator size, perhaps that could be implemented in OT.
as the + and - are already used for the viewport zoom, perhaps a modifier key could be used, such as Ctrl, Shift, whichever... together with the + and - keys.

**Comment by artisteacher** [comment]
Personally the size doesn't bother me. I do think that sometimes the pink color blends in too much - especially if there is a background at a similar value. The widget might benefit from being two-toned, like how the selection box handles were changed in #2804.

**Comment by zarco1794** [comment]
The "shift" command could be added for greater configuration accuracy: if I select "All" and want to rotate the image, pressing "shift" at the same time to make the rotation slower, and have more precision

**Comment by shun-iwasawa** [comment]
@zarco1794 

> The "shift" command could be added for greater configuration accuracy: if I select "All" and want to rotate the image, pressing "shift" at the same time to make the rotation slower, and have more precision

This feature is planned to be introduced. However I'm thinking to use the "Alt" modifier for it.
When using the tool with the Position mode, "Shift" is already occupied for moving along vertical / horizontal axis and "Ctrl" is for moving Z value.

**Comment by RodneyBaker** [comment]
Reference:  #1886

**Comment by zarco1794** [comment]
maybe this size
![1](https://user-images.githubusercontent.com/39812147/67607758-fe2da680-f75b-11e9-8285-96f120b3ec5b.JPG)

size actuallity
![arrgnzo](https://user-images.githubusercontent.com/39812147/67607862-56fd3f00-f75c-11e9-8146-e8ecbe8e36e0.jpg)

**Comment by artisteacher** [comment]
#2858 definitely helps to make more precise adjustments. However, it doesn't help in cases where it's hard to see the animate tool widget. The widget size seems fine for macOS. I'd rather see either an increase in the widget thickness, or adding a contrasting outline in strategic places.

**Comment by ave4** [comment]
This tool is very slow for me under linux, which is impossible to work with. It seems to me that these buttons are sources of problems.
https://github.com/opentoonz/opentoonz/issues/2648

![Снимок экрана от 2019-11-11 10-06-06](https://user-images.githubusercontent.com/35303790/68562306-e730c900-046a-11ea-8695-cb37b5ead6e8.png)
